scripts/vcpkg/get_mumble_dependencies.ps1

```diff
@@ -6,29 +6,29 @@ $mumble_deps = "qt5-base",
                "qt5-tools",
                "grpc",
                "boost-accumulators",
-               "opus",

```

Comment:
Why did you remove `opus` and how is that related to fixing the issue at hand? :thinking:

Reply:
We are using a submodule to deliver opus. I can add it back if needed, but it's just occupying space.

Reply:
True ^^

This change is however unrelated to the actual fix, so I'd prefer having that in a separate commit :)

Reply:
This has been done.